index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
```

Comment:
shouldn't we have a meta description and a favicon, too?

Reply:
Meta description I will add, but why do I need a favicon?